androidApp/src/main/java/com/mbta/tid/mbta_app/android/map/HomeMapView.kt

```diff
@@ -98,7 +98,7 @@ fun HomeMapView(
     var railRouteLineData: List<RouteLineData>? by remember { mutableStateOf(null) }
     var stopSourceData: FeatureCollection? by remember { mutableStateOf(null) }
 
-    val now = timer(updateInterval = 10.seconds)
+    val now = timer(updateInterval = 300.seconds)
```

Comment:
I noticed this was causing lag / jankiness and saw the[ iOS value is 300 seconds](https://github.com/mbta/mobile_app/blob/98c275c218017a428aa41951a3420eed706f25cb/iosApp/iosApp/Pages/Map/HomeMapView.swift#L46), so updated to match.

This can be further improved by moving `getAlertsByStop` to a background thread for calculation since the calculation alone takes ~1 second. That wasn't straight forward to do within the `remember` func though, and would probably need to be moved into the ViewModel. I can make a separate ticket for that